typescript/infra/config/environments/mainnet3/warp/checkWarpDeploy.ts

```diff
@@ -0,0 +1,13 @@
+import { CheckWarpDeployConfig } from '../../../../src/config/funding.js';
+import { environment } from '../chains.js';
+
+export const checkWarpDeployConfig: CheckWarpDeployConfig = {
+  docker: {
+    repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
+    tag: '36f7e14-20240823-160646',
```

Comment:
Will set to latest in a follow up, as some of the changes in this PR are needed for the image